test/e2e/webhelp.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { expect, test } from '@playwright/test';
+import { testSelector } from '../utils';
+import { WebHelpPage } from '../page/webhelp-page';
+import {  RESOLUTIONS } from '../visual/visual-constants';
+
+test.describe('WebHelp page interactivity', async () => {
+    test.beforeEach(async ({ page }) => {
+        const webHelpPage = new WebHelpPage(page, '/docs/test-page.html');
+        await webHelpPage.init();
+    });
+
+    test(`Should open page  item  properly on desktop`, async ({ page }) => {
+        await page.setViewportSize(RESOLUTIONS[0]);
```

Comment:
Would be great to change RESOLUTIONS[0], to something like "RESOLUTIONS.MOBILE"

Reply:
✅